.github/workflows/test.yml

```diff
@@ -0,0 +1,102 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  check-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
+
+    - name: check docs
+      run: |
+        echo "Making sure docs are updated"
+        make docs
+        if [[ -n $(git status --porcelain) ]]; then echo "Please update the docs using make docs"; exit 1; fi
+
+  mm-govet:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
+
+    - name: check lints
+      run: |
+        echo "Installing mattermost-govet"
+        go env GOPATH
+        go install github.com/mattermost/mattermost-govet@master
+        make gofmt govet
+
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: mmctl
+
+    - name: Checkout mm-server
+      uses: actions/checkout@v2
+      with:
+        repository: mattermost/mattermost-server
+        path: mattermost-server
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
+
+    - name: Setup mattermost-server
+      working-directory: mattermost-server
+      run: |
+        git checkout $GITHUB_HEAD_REF || git checkout master
+
+    - name: Start containers
+      working-directory: mattermost-server/build
+      env:
+        COMPOSE_PROJECT_NAME: "actions"
+      run: |
+        # cd ../mattermost-server/build
```

Comment:
redundant

Reply:
this actually is a left over thanks for the catch